packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -420,6 +420,9 @@ module.exports.subscriptionsCustomerValidator = isA.object({
   payment_type: isA.string().optional(),
   paypal_payment_error: isA.string().optional(),
   brand: isA.string().optional(),
+  billing_agreement_id: isA
+    .alternatives(isA.string(), isA.any().allow(null))
```

Comment:
Isn't `isA.string()` a subset of `isA.any()`?  Do we need `isA.string()`?

Reply:
Here I was following the patterns throughout the file. I believe this is the way you write out `string | null`